med-web-app-front/src/services/pipelinejob.service.js

```diff
@@ -12,6 +12,10 @@ class PipelineJobService {
     sendRequestForPipelineJob(username, pipelineId, fileId) {
         return axios.post(API_URL + username, {pipelineId, fileId},{ headers: authHeader() });
     }
+
+    deletePipelineJob(username, pipelineJobId, fileId) {

```

Comment:
Codacy has a fix for the issue: [Replace `··deletePipelineJob(username,·pipelineJobId,·fileId)·{␍` with `}⏎⏎··deletePipelineJob(username,·pipelineJobId,·fileId)·{`](https://app.codacy.com/gh/danielsheh02/Medical-Web-App/pullRequest?prid=9534713)
```suggestion
  }
```